py-polars/tests/parametric/test_groupby_rolling.py

```diff
@@ -17,10 +18,12 @@
 
 
 @given(
-    period=st.timedeltas(min_value=timedelta(microseconds=0)).map(
-        _timedelta_to_pl_duration
-    ),
-    offset=st.timedeltas().map(_timedelta_to_pl_duration),
+    period=st.timedeltas(
+        min_value=timedelta(microseconds=0), max_value=timedelta(days=1000)
+    ).map(_timedelta_to_pl_duration),
+    offset=st.timedeltas(
+        min_value=timedelta(microseconds=0), max_value=timedelta(days=1000)
+    ).map(_timedelta_to_pl_duration),
```

Comment:
I'm tightening the strategy here so that there aren't overflow errors in the tests

This results in a greater number of valid test cases, which helps discover even edgier cases like the one linked

I ran this locally with `max_examples=10_000`, and everything passed. So, all the edge cases might be addressed now? 🤞